packages/engine/Source/Widget/CesiumWidget.js

```diff
@@ -661,6 +841,104 @@ Object.defineProperties(CesiumWidget.prototype, {
       }
     },
   },
+
+  /**
+   * Gets or sets whether or not data sources can temporarily pause
+   * animation in order to avoid showing an incomplete picture to the user.
+   * For example, if asynchronous primitives are being processed in the
+   * background, the clock will not advance until the geometry is ready.
+   *
+   * @memberof CesiumWidget.prototype
+   *
+   * @type {boolean}
+   */
+  allowDataSourcesToSuspendAnimation: {
+    get: function () {
+      return this._allowDataSourcesToSuspendAnimation;
+    },
+    set: function (value) {
+      this._allowDataSourcesToSuspendAnimation = value;
+    },
+  },
+
+  /**
+   * Gets or sets the Entity instance currently being tracked by the camera.
+   * @memberof CesiumWidget.prototype
+   * @type {Entity | undefined}
+   */
+  trackedEntity: {
+    get: function () {
+      return this._trackedEntity;
+    },
+    set: function (value) {
+      if (this._trackedEntity !== value) {
+        this._trackedEntity = value;
+
+        //Cancel any pending zoom
+        cancelZoom(this);
+
+        const scene = this.scene;
+        const sceneMode = scene.mode;
+
+        //Stop tracking
+        if (!defined(value) || !defined(value.position)) {
+          this._needTrackedEntityUpdate = false;
+          if (
+            sceneMode === SceneMode.COLUMBUS_VIEW ||
+            sceneMode === SceneMode.SCENE2D
+          ) {
+            scene.screenSpaceCameraController.enableTranslate = true;
+          }
+
+          if (
+            sceneMode === SceneMode.COLUMBUS_VIEW ||
+            sceneMode === SceneMode.SCENE3D
+          ) {
+            scene.screenSpaceCameraController.enableTilt = true;
+          }
+
+          this._entityView = undefined;
+          this.camera.lookAtTransform(Matrix4.IDENTITY);
+        } else {
+          //We can't start tracking immediately, so we set a flag and start tracking
+          //when the bounding sphere is ready (most likely next frame).
+          this._needTrackedEntityUpdate = true;
+        }
+
+        this._trackedEntityChanged.raiseEvent(value);
+        this.scene.requestRender();
+      }
+    },
+  },
+
+  /**
+   * Gets the event that is raised when the tracked entity changes.
+   * @memberof CesiumWidget.prototype
+   * @type {Event}
+   * @readonly
+   */
+  trackedEntityChanged: {
+    get: function () {
+      return this._trackedEntityChanged;
+    },
+  },
+
+  /**
+   * Gets or sets the data source to track with the viewer's clock.
```

Comment:
```suggestion
   * Gets or sets the data source to track with the widget's clock.
```

I'll comment on a couple other instances of this but please do a pass yourself to make sure code copied over refers to a widget not a viewer in this file

Reply:
Copy paste hell!
FWIW, there were some viewer's references prior to my PR. I fixed them also...